packages/backup-restore-operator/charts/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+appVersion: v0.0.1-rc4
+description: Helm chart for installing the rancher backup-restore-operator
+name: backup-restore-operator
+version: 0.0.1
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/namespace: cattle-resources-system
```

Comment:
nit: is there any reason why we chose `cattle-resources-system`? I would have assumed the namespace here should be `cattle-backup-restore-system` or something similar that is tied to the chart name

Reply:
we chose `cattle-resources-system` as namespace and `resources` as apiversion because it was discussed in the design meeting there's a chance we can use ResourceSet CRD for cleaning up CRDs of uninstalled charts.